lf_haa_control:
  type: effort_controllers/JointPositionController
  joint: lf_haa_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}
lf_hfe_control:
  type: effort_controllers/JointPositionController
  joint: lf_hfe_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}
lf_kfe_control:
  type: effort_controllers/JointPositionController
  joint: lf_kfe_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}
lh_haa_control:
  type: effort_controllers/JointPositionController
  joint: lh_haa_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}
lh_hfe_control:
  type: effort_controllers/JointPositionController
  joint: lh_hfe_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}
lh_kfe_control:
  type: effort_controllers/JointPositionController
  joint: lh_kfe_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}
rf_haa_control:
  type: effort_controllers/JointPositionController
  joint: rf_haa_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}
rf_hfe_control:
  type: effort_controllers/JointPositionController
  joint: rf_hfe_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}
rf_kfe_control:
  type: effort_controllers/JointPositionController
  joint: rf_kfe_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}
rh_haa_control:
  type: effort_controllers/JointPositionController
  joint: rh_haa_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}
rh_hfe_control:
  type: effort_controllers/JointPositionController
  joint: rh_hfe_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}
rh_kfe_control:
  type: effort_controllers/JointPositionController
  joint: rh_kfe_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}
